Utils/Elasticsearch/tools/daily/es_backup

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+base_dir=/var/es
+data_dir=data
+backup_dir=$base_dir/backup
+
+
+[ -d $backup_dir ] \
+  || mkdir -m 0700 $backup_dir \
+  || exit 1
+
+backup_name=$backup_dir/`date +'%Y%m%d'`
+
+# Check that today's backup does not exist
+[ -f ${backup_name}.tar.gz ] \
+  && { echo "Today's backup for ES data already exists:" \
+            "manually remove it to create a new one" \
+            "(${backup_name}.tar.gz)." >&2 ;
+       exit 0;
+     }
+
+# Create today's backup file
+tar -C $base_dir -zcf ${backup_name}.tar.gz $data_dir
+EXIT_CODE=$?
+
+[ $EXIT_CODE -ne 0 ] \
+  && { echo "Failed to create backup file for ES data ($EXIT_CODE)." >&2 ;
+       exit $EXIT_CODE;
+     }
+
+# Create weekly/monthly/yearly copy
+[ `date +%u` -eq 7 ] && suffix='w'
+[ `date +%e` -eq 1 ] && suffix='m'
+[ `date +%_j` -eq 1 ] && suffix='y'
+[ -n "$format" ] && ln ${backup_name}.tar.gz ${backup_name}${suffix}.tar.gz
```

Comment:
@IATkachenko,
> Why you are using hard links instead of cp or mv?

Not `mv`: I want to have today's backup available as `20191204.tar.gz`, no matter if it was created on Sunday, first day of a month or a year.
Not `cp`: I don't want to increase disk usage.
And also not soft links, since I want to keep weekly snapshot even if it was removed as daily.

Although I could use `mv` + soft link, but would it be better?